submit-web/src/components/Submission/DocumentRow/index.tsx

```diff
@@ -55,7 +55,7 @@ export default function DocumentRow({
   return (
     <>
       <TableRow sx={{ "& > *": { borderBottom: "unset" } }}>
-        <SubmitTableCell>
+        <SubmitTableCell colSpan={10}>
```

Comment:
the colSpan usage in these changes is not the right way. there are for sure not 10 columns so why colSpan 10?